### DISCUSSION TITLE
Steps for buying Pro plugin version (and other plug ins by MusieDestiny)

### DISCUSSION THREAD

**Discussion by Dcrizz** [MuiseDestiny/zotero-style#1057] [Q&A]
Hello

I would like to use some of the functions of the pro version of Zotero style, mainly the connected papers and bibliography functions, I have tried figuring out how to purchase it off Bilibili but cant seem to do it. 

I am in the UK, would anyone be kind enough to explain the steps I would need to take to purchase it from overseas or any additional criteria I would require (e.g. temporary Chinese phone number?)  #proversionpurchase

**Comment by Marshal-Duan** [comment]
you need to use alipay, wechat pay, or QQ pay to buy pro version. all app are from China mainland.
